### PR TITLE
Document timeout units

### PR DESCRIPTION
**Pull request by anishkny**

### CODE REVIEW

**Comment by sindresorhus** [comment]
This is not entirely correct: https://github.com/avajs/ava/blob/master/docs/07-test-timeouts.md

**Comment by anishkny** [comment]
Oh didn't realize that. How about something like this:

```
    --timeout, -T           Set global timeout (millis or human-readable, e.g. 10s, 2m)
```

File: docs/05-command-line.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ $ npx ava --help
     --match, -m             Only run tests with matching title (Can be repeated)
     --update-snapshots, -u  Update snapshots
     --fail-fast             Stop after first test failure
-    --timeout, -T           Set global timeout
+    --timeout, -T           Set global timeout (millis or human-readable, e.g. 10s, 2m)
```
Comment (sindresorhus) [inline docs/05-command-line.md]:
```suggestion
    --timeout, -T           Set global timeout (milliseconds or human-readable, e.g. 10s, 2m)
```

**Comment by sindresorhus** [comment]
You also need to update the actual CLI help.

**Comment by anishkny** [comment]
> You also need to update the actual CLI help.

Done!

**Comment by novemberborn** [comment]
Lovely, thanks @anishkny!

**Pull request merged**